src/App/backend/test/Altinn.App.SourceGenerator.Integration.Tests/gen/Altinn.App.Analyzers/Altinn.App.Analyzers.FormDataWrapperGenerator/SkjemaFormDataWrapper.g.cs

```diff
@@ -171,6 +171,292 @@ int offset
+        int offset,
+        object? value
+    )
+    {
+        if (model is null || offset == -1)
+        {
+            return false;
+        }
+
+        var segment = ParseSegment(path, offset, out int nextOffset, out int literalIndex);
+        return segment switch
+        {
+            "skjemanummer" when nextOffset is -1 && literalIndex is -1 => TrySetValue<string>(val => model.Skjemanummer = val, value),
+            "skjemaversjon" when nextOffset is -1 && literalIndex is -1 => TrySetValue<string>(val => model.Skjemaversjon = val, value),
+            "skjemainnhold" => SetRecursive_WithListCreation_Altinn_App_SourceGenerator_Integration_Tests_Models_Skjema_Skjemainnhold(model, path, literalIndex, nextOffset, value),
+            "eierAdresse" when literalIndex is -1 => SetRecursive_WithObjectCreation_Altinn_App_SourceGenerator_Integration_Tests_Models_Skjema_EierAdresse(model, path, nextOffset, value),
+            _ => false,
+        };
+    }
+
+    private static bool SetRecursive_WithListCreation_Altinn_App_SourceGenerator_Integration_Tests_Models_Skjema_Skjemainnhold(
+        global::Altinn.App.SourceGenerator.Integration.Tests.Models.Skjema model,
+        global::System.ReadOnlySpan<char> path,
+        int literalIndex,
+        int nextOffset,
+        object? value
+    )
+    {
+        if (model.Skjemainnhold is null)
+        {
+            try
+            {
+                model.Skjemainnhold = new global::System.Collections.Generic.List<global::Altinn.App.SourceGenerator.Integration.Tests.Models.SkjemaInnhold?>();
+            }
+            catch
+            {
+                return false;
+            }
+        }
+        return SetRecursive(model.Skjemainnhold, path, literalIndex, nextOffset, value);
+    }
+
+    private static bool SetRecursive_WithObjectCreation_Altinn_App_SourceGenerator_Integration_Tests_Models_Skjema_EierAdresse(
+        global::Altinn.App.SourceGenerator.Integration.Tests.Models.Skjema model,
+        global::System.ReadOnlySpan<char> path,
+        int nextOffset,
+        object? value
+    )
+    {
+        if (model.EierAdresse is null)
+        {
+            try
+            {
+                model.EierAdresse = global::System.Activator.CreateInstance<global::Altinn.App.SourceGenerator.Integration.Tests.Models.Adresse>();
+            }
+            catch
+            {
+                return false;
+            }
+        }
+        return SetRecursive(model.EierAdresse, path, nextOffset, value);
+    }
+
+    private static bool SetRecursive(
+        global::System.Collections.Generic.List<global::Altinn.App.SourceGenerator.Integration.Tests.Models.SkjemaInnhold?>? model,
+        global::System.ReadOnlySpan<char> path,
+        int literalIndex,
+        int offset,
+        object? value
+    )
+    {
+        if (model is null || literalIndex < 0)
+        {
+            return false;
+        }
+
+        // Create list elements if index is out of bounds
```

Comment:
I would expect index out of bounds would result in a failed write, but I certainly see the appeal of being able to use this to just initialise 5 rows. Adding only a single element and then throwing out of bounds, if that was not enough seems inappropriate.